### PR TITLE
fix(events): use shorter prefix

### PR DESCRIPTION
**Pull request by serge1peshcoff**
shortening the API prefix for events, the frontend part.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ module.exports = {
     'no-prototype-builtins': 'off',
     'quote-props': 'off',
     'arrow-body-style': 'off',
-    'import/no-extraneous-dependencies': 'off'
+    'import/no-extraneous-dependencies': 'off',
+    'dot-notation': 'off'
```
Comment (WikiRik) [inline .eslintrc.js:43]: Just out of curiosity, why do you want to disable this rule?

Comment (WikiRik) [reply .eslintrc.js:43]: Instead of changing `this.services['events']` to `this.services.events`

Comment (serge1peshcoff) [reply .eslintrc.js:43]: I personally like this one more, but I can go with both. But anyway I don't see a point in this rule, it restricts you too much IMO

**Pull request merged**